packages/manager/fragments/navbar/src/notifications.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { emit, listen } from '@ovh-ux/ufrontend/communication';
+import style from './navbar.scss';
+
+function NavbarNotifications() {
+  const { t } = useTranslation();
+  const [notificationsCount, setNotificationsCount] = useState(0);
+
+  useEffect(() => {
+    listen(({ id, count }) => {
+      if (id === 'ovh.notifications.count') {
+        setNotificationsCount(count);
+      }
+    });
+  }, []);
+
+  return (
+    <button
+      role="button"
+      className={`oui-navbar-link oui-navbar-link_icon oui-navbar-link_tertiary ${style.navbarLink}`}
```

Comment:
I don't see `@ovh-ux/ui-kit` as a dependency so those classes are not of any use

Reply:
You're right, it's a known issue we have, we don't want to bundle the whole ui-kit in the fragment but on the other hand we are not able to import only the needed css parts of the ui-kit components (using scoped imports).
Talked about it with @cbourgois, we choosed that we would temporarily use the ui-kit classes in the fragments, knowing that the ui-kit will be imported by applications and perform a refactoring later when the ui-kit provides more modularity. Do you have any opinion on it ?

Reply:
I understand and I agree that we shouldn't import the ui-kit yet in fragments to avoid importing things we don't need. But maybe the ui-kit could be set as a peer, just so the apps using the fragment knows they have to bring it in order to make the fragment work properly :)

Reply:
added the peer dep